pkg/trace/obfuscate/obfuscate.go

```diff
@@ -44,6 +45,26 @@ type Config struct {
 
 	// Redis enables obfuscatiion of the "memcached.command" tag for spans of type "memcached".
 	Memcached bool
+
+	// literalEscapes reports whether we should treat escape characters literally or as escape characters.
+	// A non-zero value means 'yes'. Different SQL engines behave in different ways and the tokenizer needs
+	// to be generic.
+	// Not safe for concurrent use.
+	literalEscapes int32
+}
+
+// SetLiteralEscapes sets whether or not escape characters should be treated literally,
+func (o *Obfuscator) SetLiteralEscapes(ok bool) {
```

Comment:
I guess we'd have to apply the same here, right?
```suggestion
func (o *Obfuscator) SetSQLLiteralEscapes(ok bool) {
```